src/apps/lwaftr/lwaftr.lua

```diff
@@ -78,6 +79,13 @@ end
 LwAftr = {}
 
 function LwAftr:new(conf)
+   -- It's a bit of a hack to deal with tagging here, but the front-ends are
+   -- rapidly evolving, and this will happen regardless of which one is used.
```

Comment:
Modifying `constants.ethernet_header_size` before `LwAftr:new()` is called does not work as expected, before other modules imported at the top of the file pick the value from the `constants` one to define their own locals, e.g.:

```lua
local constants = require("apps.lwaftr.constants")
local header_size = constants.ethernet_header_size + some_other_size
```

The locals in modules derived from `constants.ethernet_header_size` in the imported modules will be using the old value even after a call to `LwAftr:new()`. We'll have to remove caching as locals the values derived from `constants.ethernet_header_size`, or figure out some other way.